src/lib/cosmos/components/index.tsx

```diff
@@ -0,0 +1,222 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { formatEther } from '@ethersproject/units'
+import { Web3ReactHooks } from '@web3-react/core'
+import { MetaMask } from '@web3-react/metamask'
+import { Network } from '@web3-react/network'
+import { Connector } from '@web3-react/types'
+import { SupportedChainId } from 'constants/chains'
+import { useAtom } from 'jotai'
+import { providerAtom } from 'lib/cosmos.decorator'
+import styled from 'lib/theme'
+import { useEffect, useState } from 'react'
+
+import { connectors } from '../connectors'
+import { hooks as metaMaskHooks, metaMask } from '../connectors/metaMask'
+import { hooks as networkHooks, network } from '../connectors/network'
+function getName(connector: Connector) {
+  if (connector instanceof MetaMask) {
+    return 'MetaMask'
+  } else if (connector instanceof Network) {
+    return 'Network'
+  } else {
+    return 'Unknown'
+  }
+}
+
+function Status({
+  connector,
+  hooks: { useChainId, useAccounts, useError },
+}: {
+  connector: Connector
+  hooks: Web3ReactHooks
+}) {
+  const chainId = useChainId()
+  const accounts = useAccounts()
+  const error = useError()
+
+  const connected = Boolean(chainId && accounts)
```

Comment:
After disconnecting from MetaMask, this is still marked true in the UI, with "Chain Id: 1", "Accounts: None".